src/test/java/com/auctionapp/api/model/CategoryTest.java

```diff
@@ -0,0 +1,18 @@
+package com.auctionapp.api.model;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.auctionapp.api.model.dto.CategoryDto;
+
+import org.junit.jupiter.api.Test;
+
+public class CategoryTest {
```

Comment:
No need to waste your time writing these. We can trust that getters and setters work. In general, if you feel that something makes no sense to test, you are likely right.